CMakeLists.txt

```diff
@@ -58,8 +58,13 @@ endif()
 if(EMSCRIPTEN)
   message("Building for Emscripten")
   set(MANIFOLD_FLAGS -fexceptions -D_LIBCUDACXX_HAS_THREAD_API_EXTERNAL -D_LIBCUDACXX_HAS_THREAD_API_CUDA)
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -sALLOW_MEMORY_GROWTH=1 -fexceptions -sDISABLE_EXCEPTION_CATCHING=0")
+  # ALLOW_MEMORY_GROWTH is slow, so we just allow 2gb of memory usage
```

Comment:
Interesting, how slow (roughly)?

Reply:
Not sure, this is a compiler warning (https://github.com/WebAssembly/design/issues/1271), and it does not hurt to make it a fixed memory size. This probably affects our js callbacks as they perform quite a lot of memory read/write.

I think a fixed memory size should be fine, it does not occupy physical memory if not that much memory is actually used, and because wasm by default is using 32 bit pointers we cannot use more than 4gb of memory anyway. Forgot why I set it to 2gb though, perhaps just a random choice.